lib/redis/commands/bitmaps.rb

```diff
@@ -27,9 +27,13 @@ def getbit(key, offset)
       # @param [String] key
       # @param [Integer] start start index
       # @param [Integer] stop stop index
+      # @param [String, Symbol] scale the scale of the offset range
+      #     e.g. 'BYTE' - interpreted as a range of bytes, 'BIT' - interpreted as a range of bites
       # @return [Integer] the number of bits set to 1
-      def bitcount(key, start = 0, stop = -1)
-        send_command([:bitcount, key, start, stop])
+      def bitcount(key, start = 0, stop = -1, scale: 'BYTE')
+        command = [:bitcount, key, start, stop]
+        command << 'BIT' if scale.to_s.upcase == 'BIT'
```

Comment:
```suggestion
        command << 'BIT' if scale.
```
I think we should append it unless it's not set. Because here if I pass `scale: "lol"`, it will silently do nothing.

With `scale: nil` in the signature.

Reply:
Agreed, thanks.